src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1220,21 +1221,28 @@ class LedgerRPC_test : public beast::unit_test::suite
     }
 
     void
-    testLedgerEntryUnknownOption()
+    testLedgerEntryInvalidParams(unsigned int apiVersion)
```

Comment:
I'm not sure what the convention is in rippled but i'd use uint32_t or perhaps uint16_t. Fine as it is if others are happy though